app/routes/u.github/$slug/welcome.tsx

```diff
@@ -0,0 +1,128 @@
+import { json, LoaderFunction, redirect } from "@remix-run/node";
+import { Form, Link, useLoaderData } from "@remix-run/react";
+import { FC } from "react";
+import TextField from "~/components/TextField";
+import { destroySession, getCurrentUser, getSession } from "~/session.server";
+import { User } from "~/types";
+import formStyles from "~/styles/forms.css";
+import Button from "~/components/Button";
+import CloseIcon from "~/components/icons/CloseIcon";
+import { getProfileRouteForUser } from "~/utils/routes";
+
+export function links() {
+  return [{ rel: "stylesheet", href: formStyles }];
+}
+
+export const loader: LoaderFunction = async ({ request }) => {
+  const session = await getSession(request.headers.get("Cookie"));
+  const currentUser = await getCurrentUser(session);
+
+  if (!currentUser) {
+    // Redirect to the login page if needed
+    session.unset("idToken");
+    return redirect("/login", {
+      headers: {
+        "Set-Cookie": await destroySession(session),
+      },
+    });
+  }
+
+  return json({ user: currentUser });
+};
+
+const Welcome: FC = () => {
```

Comment:
Here is our "Welcome" modal, which users see the first time they log in to OSH. Using the file name, we can figure out the page's path: `/u/github/:slug/welcome`.

If you recall the previous file in the tour, it lived at `/u/github/:slug` and had an `<Outlet />` component. Thanks to nested routing, Remix will render this component _inside_ of the previous file. 😮